### PR TITLE
Various fixes/updates

### PR DESCRIPTION
**Pull request by scoot404**
* Info for promos sourced from Pokebeach/Pokellector.
* Fixes pioIDs on Champion's Path/Vivid Voltage since they weren't correct.
* Legality based on 11/27 rulings (new promos for Standard and bans for expanded).
* Adds Vivid Voltage to standard/expanded.
* Fixes missing SM promos for standard 2021 in formats.yaml
* Marked missing Prism Stars that were from Team Up
* Fixed the text on the Special Energy from Vivid Voltage
* Adds line breaks to Darkness Ablaze special energy

### CODE REVIEW

File: src/main/resources/cards/429-sword_shield_promos.yaml [hunk 1]
```diff
@@ -1406,4 +1406,308 @@ cards:
...
+  weaknesses:
+  - type: R
+    value: x2
+  resistances:
+  - type: G
+    value: '-30'
+  rarity: Promo
+- id: 429-SWSH077
+  pioId: swsh-SWSH077
+  enumId: ZAMAZENTA_V_SWSH76
```
Comment (flagrama) [inline src/main/resources/cards/429-sword_shield_promos.yaml:1666]:
```suggestion
  enumId: ZAMAZENTA_V_SWSH77
```

**Comment by flagrama** [comment]
Are images available and uploaded for these new promos?

**Comment by scoot404** [comment]
> 
> 
> Are images available and uploaded for these new promos?

Maybe? I'll go ahead and upload whatever's missing, I'll take a look in the server in a bit.

**Review by flagrama — APPROVED** [review]
Yeah, I uploaded the ones that actually have images available, so only one missing is the Special Delivery Charizard. Looks good to me.

**Pull request merged**